layouts/community/60_ansi/stanrc85-ansi/keymap.c

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2018 MechMerlin
```

Comment:
=)

Reply:
Shhh :D

Reply:
Just ignore that part :)

Reply:
Ideally, you'd want to add your name, or change the name here. 

If you're fine with leaving this, and @mechmerlin doesn't mind, then we can leave it as is.